pypeit/images/processrawimage.py

```diff
@@ -13,6 +13,10 @@
 from IPython import embed
 
 
+# TODO JFH This routine needs to be reworked to always return images in counts, and to always return a noise model.
+# It is completely wrong to ever do anything with images in ADU.
```

Comment:
This was chosen to deal with saturation, e.g. arcs.
If we modify here, make sure to deal with saturation
throughout the calibration codes.

Reply:
I haven't changed anything here, just noting that I think we should change this in the future. The problem with ADU is then you don't have a noise model, and sometimes we need a noise model even for calibration frames.

Reply:
The other problem is that some codes are masking images assuming they are in ADU, when they are actually multiplied by the gain (like flats).